docs/content/faq/operators.mdx

```diff
@@ -159,7 +159,7 @@ It sounds like you didn't wipe your `data` folder? Turn off your node, remove th
 
 ## What errors are acceptable to see when my node is running?
 
-No errors should be considered acceptable. If there are ones that are constantly recurring, please bring it to our attention, and if we determine that they are not truly errors, we will adjust the logging to reflect.
+No errors should be considered acceptable. If there are errors that are constantly recurring, please bring it to to the core team's attention so logging can be adjusted accordingly, if required.
```

Comment:
let's avoid saying "core team's attention". we don't qualify who is part of the team or how to reach them from what I can tell. I'd suggest we say they join our Discord and ask questions in a channel for node operators. My PR will remove references to "us" or "we" or "core team" as well